examples/grafana.build.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
+
+#
+# A rkt container pod for running a Statsd/Graphite/Grafana metrics stack.
+#
+# Start the containers with new data storage directory:
+#
+#   mkdir data
+#   sudo rkt run \
+#     --port=carbon:2003 \
+#     --port=statsd:8125 \
```

Comment:
As I'm still a total graphite noob, what role does `statsd` play in all of this?